aquadrone_pneumatics/src/aquadrone_pneumatics.py

```diff
@@ -0,0 +1,49 @@
+import rospy
+from aquadrone_pneumatics.msg import command, status
+from aquadrone_pneumatics.srv import PneumaticsCommands, PneumaticsCommandsResponse
+import RPi.GPIO as GPIO
+
+
+command_to_pin = {
+    "l_torpedo": [6, 0.05],
+    "r_torpedo": [7, 0.05],
+    "claw_open": [5, 5.0],
+    "claw_close": [5, 5.0],
+}
+
+class Pneumatics():
+    def __init__(self):
+        rospy.init_node('pneumatics_server', log_level=rospy.DEBUG)
+        s = rospy.Service('pneumatics_commands', PneumaticsCommands, self.handle_pneumatics_commands)
+        self.claw_open = False
+        self.l_torpedo_fired = False
+        self.r_torpedo_fired = False
+
+    def handle_pneumatics_status(self):
+        msg = PneumaticsCommandsResponse()
+        msg.claw_open = self.claw_open
+        msg.l_torpedo_open = self.l_torpedo_fired
+        msg.r_torpedo_open = self.r_torpedo_fired
+        return msg
+
+    def fire_torpedo(self, msg):
+        GPIO.output(command_to_pin[msg][0], GPIO.HIGH)
```

Comment:
We should create a simulated pneumatics node as well. The simulated version will behave the same, just without sending any commands via the GPio pins. This will let us have a version that we can run in the simulated environment.

Reply:
e878a21